win32/src/winapi/kernel32/resource.rs

```diff
@@ -77,18 +81,18 @@ pub fn find_resource<'a>(
     hInstance: HINSTANCE,
     typ: ResourceKey<&Str16>,
     name: ResourceKey<&Str16>,
-) -> Option<&'a [u8]> {
+) -> Option<Range<usize>> {
     let image = mem.slice(hInstance..);
     if hInstance == kernel32.image_base {
         let section = kernel32.resources.as_slice(image)?;
-        Some(&image[pe::find_resource(section, typ.into_pe(), name.into_pe())?])
+        pe::find_resource(section, typ.into_pe(), name.into_pe())
```

Comment:
Now that I'm looking, I wonder if pe::find_resource ought to return a Range<u32> as well, hm.  If you look at its impl you can see it has u32s and it only casts to usize on the way out.